### PR TITLE
Add a quote by Johann Wolfgang von Goethe

### PR DESCRIPTION
**Pull request by mkhy19**
Added a quote by Johann Wolfgang von Goethe.

## ✅ New Quote Checklist
- [x] The quote(s) added are in alphabetical order by author's first name and not ❌ duplicates.
- [x] The quote(s) added are about **learning a foreign language**.
- [x] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [x] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
This is a [duplicate quote](https://github.com/jayehernandez/letra-extension/blob/08226428a914bb75266e566bad4ad86056bab92e/server/data/quotes.json#L403) and doesn't align with our contribution guidelines. Please update it.

**Comment by mkhy19** [comment]
I updated it @Kvaibhav01

File: server/data/quotes.json [hunk 1]
```diff
@@ -56,8 +56,8 @@
     "sentence": "One of the benefits of being bicultural is simply the awareness that how you live is not the only way."
   },
   {
-    "author": "Johann Wolfgang von Goethe",
-    "sentence": "He who knows no foreign languages knows nothing of his own."
+    "author": "George Carlin",
+    "sentence": "“Meow” means “woof” in cat.”"
```
Comment (Kvaibhav01) [inline server/data/quotes.json:60]: It doesn't match the contribution requirement. We need quotes regarding "learning a foreign language" or similar. You need to change it.

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]

**Comment by mkhy19** [comment]
It was hard to find a quote about learning a language but finally I found it LOL, Hope you accept it.

File: server/data/quotes.json [hunk 2]
```diff
@@ -56,8 +56,8 @@
     "sentence": "One of the benefits of being bicultural is simply the awareness that how you live is not the only way."
   },
   {
-    "author": "George Carlin",
-    "sentence": "“Meow” means “woof” in cat.”"
+    "author": "Dutch Proverb",
```
Comment (Kvaibhav01) [inline server/data/quotes.json:59]: Please add it alphabetically as others are. Read the [contribution guidelines checklist](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_quote.md#-checklist) carefully before submitting the PR.

File: server/data/quotes.json [hunk 3]
```diff
@@ -56,8 +56,8 @@
...
+    "sentence": "Who knows the language is at home everywhere.”"
```
Comment (Kvaibhav01) [inline server/data/quotes.json:60]: No need of adding another quote symbol at the end of the sentence.

```suggestion
    "sentence": "Who knows the language is at home everywhere."
```

**Review by Kvaibhav01 — COMMENTED** [review]
Update as required.

**Comment by mkhy19** [comment]
DONE @Kvaibhav01

**Comment by Kvaibhav01** [comment]
It's still not correct. You need to arrange it alphabetically with the author's name as written in the guidelines.

**Comment by mkhy19** [comment]
Done again, LOL

**Comment by jayehernandez** [comment]
Thank you for contributing to Letra ☀️, @mkhy19!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Pull request merged**